generator/internal/connection_generator.cc

```diff
@@ -51,10 +51,11 @@ Status ConnectionGenerator::GenerateHeader() {
        vars("retry_traits_header_path"), "google/cloud/backoff_policy.h",
        "google/cloud/connection_options.h",
        HasLongrunningMethod() ? "google/cloud/future.h" : "",
-       HasPaginatedMethod() ? "google/cloud/internal/pagination_range.h" : "",
        HasLongrunningMethod() ? "google/cloud/polling_policy.h" : "",
        "google/cloud/status_or.h",
-       HasStreamingReadMethod() ? "google/cloud/stream_range.h" : "",
+       HasStreamingReadMethod() || HasPaginatedMethod()
+           ? "google/cloud/stream_range.h"
```

Comment:
Note, we'll still need to include `google/cloud/internal/pagination_range.h` in any file that calls the `MakePaginationRange()` function.